27_advanced-unsupervised-ml.qmd

```diff
@@ -656,6 +656,7 @@ library(censusapi)
 
 sahie <- getCensus(
   name = "timeseries/healthins/sahie",
+  key = Sys.getenv("CENSUS_KEY"),
```

Comment:
This shouldn't be necessary. You should be able to install the key like with `library(tidycensus)`.